src/motion/motion.ts

```diff
@@ -114,7 +114,9 @@ export class Motion implements vscode.Disposable {
      * space at the provided position in a lighter color.
      */
     private highlightBlock(start: Position): void {
+        start.positionOptions = PositionOptions.CharacterWiseInclusive;
```

Comment:
This still feels hacky.

Can you change line 118 to `this.highlightRange(start, start.translate(0, 1));` and then that should fix the bug.